.github/workflows/pypi.yml

```diff
@@ -0,0 +1,102 @@
+name: PyPI
+on:
+  push:
+    branches:
+      - master
+      - auto-release
```

Comment:
Do we want this to run on every merge to master? And what is the `auto-release` branch?

Reply:
`auto-release` is a branch I made in my fork. I was testing the feasibility of this auto-release workflow, so made it run on every merge to master.

Reply:
Ok sure. That will need removing before merging though.